File: common/src/main/java/com/pluxity/label3d/Label3DCreateRequest.java
package com.pluxity.label3d;

import com.pluxity.feature.entity.Spatial;

public record Label3DCreateRequest(
        String id,
        String displayText,
        Long facilityId,
        String floorId,
        Spatial position,
        Spatial rotation,
        Spatial scale) {}

File: common/src/main/java/com/pluxity/label3d/Label3DCreateRequest.kt
package com.pluxity.label3d

import com.pluxity.feature.entity.Spatial

data class Label3DCreateRequest(
    val id: String,
    val displayText: String?,
    val facilityId: Long?,
    val floorId: String?,
    val position: Spatial?,
    val rotation: Spatial?,
    val scale: Spatial?,
)
